scarb/tests/build_targets.rs

```diff
@@ -184,6 +184,7 @@ fn compile_dep_not_a_lib() {
         .assert()
         .failure()
         .stdout_matches(indoc! {r#"
+            warn: hello v1.0.0 ([..]) ignoring invalid dependency `dep` which is missing a lib or cairo-plugin target
```

Comment:
hmm why did this line appear? are we able to avoid it?

Reply:
We are now pulling deps for each generated compilation unit https://github.com/software-mansion/scarb/blob/cc3ff3894d6aef6d8a4d5d0323a103a6bff8cede/scarb/src/ops/resolve.rs#L317

Reply:
discussed offline